src/main/java/io/antmedia/AntMediaApplicationAdapter.java

```diff
@@ -1296,7 +1359,27 @@ public Result startStreaming(Broadcast broadcast)
 				broadcast.getType().equals(AntMediaApplicationAdapter.STREAM_SOURCE) ||
 				broadcast.getType().equals(AntMediaApplicationAdapter.VOD)
 				)  {
-			result = getStreamFetcherManager().startStreaming(broadcast);
+
+			if(isClusterMode()){
+				String broadcastOriginAddress = broadcast.getOriginAdress();
+
+				if(broadcastOriginAddress.equals(getServerSettings().getHostAddress())){
```

Comment:
broadcastOriginAddress can be null or empty.  It may throw NullPointerException. Just reverse check

I mean we know `getServerSettings().getHostAddress()` cannot be nullable. So 

`getServerSettings().getHostAddress().equals(broadcastOriginAddress)` does not cause NullPointerException and no need to have `broadcastOriginAddress != null` check

Reply:
done